### ISSUE TITLE
Task should allow a empty "Assign To" field

### ISSUE THREAD

**Issue by sadortun** [orocrm/crm#184]

**Comment by artml** [comment]
@sadortun Makes sense. Added to our backlog, but can't promise any implementation timeline yet, so I'm leaving it as open to community as well.

As an interim solution, you can assign them to yourself while you're looking for assignee.

**Comment by sadortun** [comment]
@artml I had a quick look in the bundle, and it seems that currently the "Assign to" is actually the owner of the task. Right ?

We should probably create a new ManyToOne to User and use the current Owner field as a "Created by" or Originator.

We have no plans to implement this yet, but it would be a nice improvement
